rasa_core/channels/socketio.py

```diff
@@ -141,11 +137,24 @@ def disconnect(sid):
             logger.debug("User {} disconnected from socketio endpoint."
                          "".format(sid))
 
+        @sio.on('session_request', namespace=self.namespace)
+        def session_request(sid, data):
+            if data is None:
+                data = {}
+            if 'session_id' not in data or data['session_id'] is None:
+                data['session_id'] = str(uuid.uuid4())
+            sio.emit("session_confirm", data['session_id'])
+            logger.debug("User {} disconnected from socketio endpoint."
```

Comment:
I guess we mean "connected"?
```suggestion
            logger.debug("User {} connected to socketio endpoint."
```